packages/driver/cypress/e2e/commands/actions/type.cy.js

```diff
@@ -1794,33 +1814,36 @@ describe('src/cy/commands/actions/type - #type', { browser: '!webkit' }, () => {
     })
 
     // https://github.com/cypress-io/cypress/issues/7088
+    // In WebKit, setting the inputType is not supported by the InputEvent constructor.
+    // This results in the inputType being unset in any of the simulated beforeinput events.
+    // https://bugs.webkit.org/show_bug.cgi?id=170416
```

Comment:
Should this limitation be updated to also mention the missing `inputType` prop? https://app.clickup.com/t/18033298/PM-1796?block=block-3bf03878-604a-44e1-84bb-e5d26fc7f749